### PR TITLE
Feat/aws support

### PR DESCRIPTION
**Pull request by lleadbet**
Adds AWS support to BAS. 

Tied to: https://github.com/apollosolutions/build-a-supergraph/pull/37

Of note: 
* Adds AWS deployment workflow.
* Modifies this subgraph to follow the service name of `subgraph-b` (`http://graphql.subgraph-a.svc.cluster.local/`).

### CODE REVIEW

File: .github/workflows/_deploy-gke.yml [hunk 1]
```diff
@@ -70,10 +70,6 @@ jobs:
           cluster_name: ${{ secrets.CLUSTER_PREFIX }}-${{ inputs.cluster_suffix }}
           location: ${{ env.GKE_ZONE }}
 
-      - name: Install Helm
```
Comment (lennyburdette) [inline .github/workflows/_deploy-gke.yml:73]: why is this removed? you didn't remove it in subgraph-b

**Review by lennyburdette — APPROVED** [review]

File: .github/workflows/_deploy-gke.yml (hunk 1 above)
Comment (lleadbet) [reply .github/workflows/_deploy-gke.yml:73]: Helm is installed by default on the GH Actions workers, so it's not needed- removed it from this repo as I missed it in a few spots.
